src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/Microsoft.Azure.ServiceBus.Tests.csproj

```diff
@@ -60,10 +49,4 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
-
```

Comment:
Just for my information is there a reason you moved the xunit.runner.json item? I just want to be sure there isn't some strange issue you are working around.

Reply:
Nope, I was just being OCD since I was in here doing manual edits.  :)